core/metacling/src/TCling.h

```diff
@@ -166,6 +166,9 @@ class TCling final : public TInterpreter {
 
    DeclId_t GetDeclId(const llvm::GlobalValue *gv) const;
 
+   Int_t DeepAutoLoadImpl(const char *cls);
+   Int_t ShallowAutoLoadImpl(const char *cls);
```

Comment:
Both functions seem to be aiding only the `TCling::AutoLoad` function and do not manipulate a lot of state in TCling. Should we just make them file static?